### PR TITLE
Chore/filmo 004 setup circleci to run remote tests

### PR DESCRIPTION
**Pull request by WinnersProx**
## Description
- configure circleci to run remote tests 

## Type of change
Please select the relevant option

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
Please describe the tests that you ran to verify your change
- [ ] End to end
- [ ] Integration
- [X] Unit testing
- [ ] N/A

## How to Manually Test
- Checkout to this branch
- Open the project in your terminal
- Run ```npm test```
- All considered test cases will be run and results will be displayed on your CLI

## Checklist:
- [X] My code follows the style guidelines of this project
- [ ] I have linted my code prior to submission
- [X] I have made corresponding changes to the documentation
- [ ] My changes generates no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by dnuwa** [comment]
@WinnersProx, please make the test you wrote pass. the project already has an endpoint that you could test. please reach out n I will walk you through how to achieve that.

**Comment by WinnersProx** [comment]
@dnuwa have you tried to run the tests?
On my side they're passing without any issue

**Comment by WinnersProx** [comment]
> @WinnersProx, please make the test you wrote pass. the project already has an endpoint that you could test. please reach out n I will walk you through how to achieve that.

Would you run the tests and show me your screenshot so that we may fix it

File: app/tests/root.test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            .set('Content-type', 'application/json')
+            .end((err, res) => {
+                if(err) done(err);
+                expect(res.status).to.equal(200);
+                expect(res.body).to.have.property('message');
+                done();
+        });
+
+    });
+});
```
Comment (dnuwa) [inline app/tests/root.test.js]: please add a new line

**Comment by dnuwa** [comment]
@WinnersProx good job making the tests pass. 
one concern though, how can we know that CircleCI is monitoring and running these tests online?

**Comment by WinnersProx** [comment]
> @WinnersProx good job making the tests pass.
> one concern though, how can we know that CircleCI is monitoring and running these tests online?

It's what i'm working on @dnuwa

**Pull request merged**